vscode-extension/package.json

```diff
@@ -150,7 +153,31 @@
           }
         ]
       }
-    ]
+    ],
+    "menus": {
+      "editor/title": [
+        {
+          "command": "vscode-aiconfig.share",
+          "when": "resourcePath =~ /.*(\\.aiconfig\\.json|\\.aiconfig\\.yaml|\\.aiconfig\\.yml|\\.aiconfig)$/",
+          "group": "navigation"
+        }
+      ],
+      "file/newFile": [
+        {
+          "command": "vscode-aiconfig.createAIConfigJSON"
+        },
+        {
+          "command": "vscode-aiconfig.createAIConfigYAML"
+        }
+      ],
+      "webview/context": [
```

Comment:
Ok so just to confirm, this is whenever you right click inside of the editor webview? https://code.visualstudio.com/api/extension-guides/webview#context-menus

If so, do you think it would also be good to have the "set API Keys" command here too?

Reply:
Yes, that would be a great command to add as well. Good idea!

Reply:
Added